js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -1700,6 +1701,13 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
         }
     },
 
+    _escapeCssSearchQuery: function(query) {
```

Comment:
'CssSearchQuery' is a very wide concept. This function will not escape any css search query for any purpose.
It can escape special characters (slash and apostrophe) in the attribute value to build the "[attrName='attrValue']" selector string (for the document.querySelector function) when the ' (apostrophe) symbol is used.